Framework/PythonInterface/mantid/dataobjects/src/Exports/PeakShapeDetectorBin.cpp

```diff
@@ -39,5 +39,5 @@ void export_PeakShapeDetectorBin() {
       .def("__init__", make_constructor(&createPeakShapeDetectorBin, default_call_policies(),
                                         (arg("detectorBinList"), arg("frame") = SpecialCoordinateSystem::None,
                                          arg("algorithmName") = "", arg("algorithmVersion") = -1)))
-      .def("__eq__", &PeakShapeDetectorBin::operator==, arg("self"), arg("other"));
```

Comment:
I see, maybe it's best to keep it as it was before? I hadn't realised that the other Peak shapes had the same behaviour, so I think they should all be consistent with each other.